Transfer/Sources/Extensions/String.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+extension String {
+    /// Format amount for currency code using users locale
+    /// - Parameter currencyCode: currency code
+    /// - Returns: a formatted amount string
+    public func formatToCurrency(with currencyCode: String?) -> String {
+        guard let currencyCode = currencyCode, !self.isEmpty
+        else { return "0" }
+        return TransferAmountCurrencyFormatter.formatCurrencyWithSymbol(self, with: currencyCode)
+    }
+}
```

Comment:
move this to Common module if you need it in every module

Reply:
Updated

Reply:
What is this logic? Is it specific to just Transfer Module that's why you added it here? Also I thought you rename TransferAmountCurrencyFormatter